test/e2e/functional/tests/e2e.go

```diff
@@ -516,69 +515,67 @@ var _ = Describe("metallb", func() {
 			Name:      "addresspool1",
 			Namespace: OperatorNameSpace,
 		}
-
+		AfterEach(func() {
+			_ = testclient.Client.Delete(context.Background(), addresspool)
+		})
 		It("should have created, update and finally delete addresspool correctly", func() {
-			By("Creating addresspool object ", func() {
-				Expect(testclient.Client.Create(context.Background(), addresspool)).Should(Succeed())
+			By("Creating addresspool object ")
+			Expect(testclient.Client.Create(context.Background(), addresspool)).Should(Succeed())
 
-				By("Checking AddressPool resource is created")
-				Eventually(func() error {
-					err := testclient.Client.Get(context.Background(), key, addresspool)
-					return err
-				}, metallbutils.Timeout, metallbutils.Interval).Should(Succeed())
+			By("Checking AddressPool resource is created")
+			Eventually(func() error {
+				err := testclient.Client.Get(context.Background(), key, addresspool)
+				return err
+			}, metallbutils.Timeout, metallbutils.Interval).Should(Succeed())
 
-				By("Checking ConfigMap is created and matches addresspool configuration")
-				assertConfigMapMatchYAML(`address-pools:
+			By("Checking ConfigMap is created and matches addresspool configuration")
+			assertConfigMapMatchYAML(`address-pools:
 - name: addresspool1
   protocol: layer2
   addresses:
   - 1.1.1.1-1.1.1.100
 `)
-			})
 
-			By("Update the same addresspool object with different range ", func() {
-				addresspool.Spec = metallbv1beta1.AddressPoolSpec{
-					Protocol: "layer2",
-					Addresses: []string{
-						"1.1.1.1-1.1.1.200",
-					},
-					AutoAssign: &autoAssign,
-				}
+			By("Update the same addresspool object with different range ")
```

Comment:
and here (how many extra spaces we have hhh)

Reply:
done + couldn't find more